src/app/mini-me/mini-me-setup/mini-me-setup.page.html

```diff
@@ -1,24 +1,45 @@
+<ion-header class="ion-no-border">
+  <ion-toolbar color="none">
+  </ion-toolbar>
+</ion-header>
 
-<ion-content>
 
-  <ion-row style="height: 64px"></ion-row>
+<!-- <ion-content>
 
-  <ion-content>
-    <ion-content class="ion-padding">
-      <ion-row style="height: 16px"></ion-row>
-      <div style="text-align:center"><h1>Set Up Mini Me</h1>
+  <ion-grid style="height: 100%;">
+    <ion-row style="background-color: aqua;">
+      <ion-col><h1 style="text-align: center; height: 25%;">Set Up Mini Me</h1>
         <ion-row style="height: 16px"></ion-row>
   
-    <p>Shall we create a Mini Me to 
+    <h3>Shall we create a Mini Me to 
       cheer for you and others in your
-      self-care journey?</p>
-      <ion-row style="height: 32px"></ion-row>
-  
-    <ion-img src="https://kidscreen.com/wp/wp-content/uploads/2018/07/Simon-Cat.jpg" alt="Simons white cat excited for you"></ion-img>
-    <ion-row style="height: 16px"></ion-row>
-  
-    <ion-row style="height: 32px"></ion-row>
-  
-    <ion-button>Yup! Let's do it</ion-button></div>
+      self-care journey?</h3>
+    </ion-col>
+    </ion-row>
+ 
+
+      <ion-row style="background-color: blue; size: 50%;"><ion-col>
+      <ion-img class="ml-7 mr-7 mb-10" src="./assets/avataaars/justina.svg"></ion-img></ion-row>
+
+  <ion-row style="background-color: aqua; height: 25%;">
+
+    <ion-button><h4 style="color: white;">Let's do it</h4></ion-button></ion-row> -->
+
+<ion-content>
+<ion-grid style="height: 100%;">
```

Comment:
Generally you want to tab this over by one (two spaces in our case) so you can quickly see where tags start and end.

Reply:
Confirmed that the swipe still works when viewing on the actual iPhone simulator. There's still the blue and teal background that is visible. Normally I would remove it before a pull request. I would approve this as a work in progress? What do you think @kaisenl ?

Reply:
Cool, sounds good. I can test on a phone after it merges. As for the colors, it's fine for now as a work in progress I think.